requirements.txt

```diff
@@ -1,12 +0,0 @@
-appdirs==1.4.3
```

Comment:
I removed it since virtualenv won't work anymore. I should probably leave and update requirements.txt for users that prefer something else than the supplied Dockerfile.